### PR TITLE
Update comparison_to_alternatives.mdx

### PR DESCRIPTION
**Pull request by gmourier**
Update of the comparative table:

- Addition of the supported compression algs
- Addition of the platform support (CPU architecture)
- Update the faceted search

### CODE REVIEW

File: learn/what_is_meilisearch/comparison_to_alternatives.mdx [hunk 1]
```diff
@@ -66,11 +66,12 @@ Can't find a client you'd like us to support? [Submit your idea or vote for it](
...
 |         | Meilisearch | Algolia | Typesense | Elasticsearch |
 |---|:---:|:----:|:---:|:---:|
-| Schemaless | ✅ | ✅ | ✅ | ✅ |
+| Schemaless | ✅ | ✅ | 🔶 <br /> `id` field is required and must be a string | ✅ |
 | Nested field support | ✅ | ✅ | ✅ | ✅ |
 | Nested document querying | ❌ | ❌ | ❌ | ✅ |
 | Automatic document ID detection | ✅ | ❌ | ❌ | ❌ |
-| Native document formats | `JSON`, `NDJSON`, `CSV` | `JSON` | `JSON`, `NDJSON` | `JSON`, `NDJSON`, `CSV` |
+| Native document formats | `JSON`, `NDJSON`, `CSV` | `JSON` | `NDJSON` | `JSON`, `NDJSON`, `CSV` |
+| Compression Support | Gzip, Deflate, Brotli | Gzip | ❌ | Gzip |
```
Comment (gmourier) [inline learn/what_is_meilisearch/comparison_to_alternatives.mdx:74]: Elasticsearch compression support to be completed

**Review by guimachiavelli — DISMISSED** [review]

File: learn/what_is_meilisearch/comparison_to_alternatives.mdx (hunk 1 above)
Comment (gmourier) [reply learn/what_is_meilisearch/comparison_to_alternatives.mdx:74]: I can only find about Gzip support so far

**Review by Kerollmops — CHANGES_REQUESTED** [review]

**Review by Kerollmops — APPROVED** [review]
I love the changes 👍

**Comment by gmourier** [comment]
Ready to get go, @guimachiavelli! 🚂 Thanks!

**Review by guimachiavelli — APPROVED** [review]

**Pull request merged**